INSTALL.md

```diff
@@ -1,82 +1,393 @@
+
+### Installing devkitPro (macOS)
+Download the `devkitpro-pacman-installer.pkg` package from [here](https://github.com/devkitPro/pacman/releases).
+
+Open the package to install devkitPro pacman.
+
+devkitPro must be configured with the tools required for GBA development. Run the following commands:
+```bash
+sudo dkp-pacman -Sy
+sudo dkp-pacman -S gba-dev
+sudo dkp-pacman -S devkitarm-rules
+```
+At this point, press Enter to confirm the installation. After the tools are installed, devkitPro must now be made accessible from anywhere by the system. To do so, run the following commands:
+
+```
+export DEVKITPRO=$HOME/devkitpro
+echo "export DEVKITPRO=$DEVKITPRO" >> ~/.bashrc
+export DEVKITARM=$DEVKITPRO/devkitARM
+echo "export DEVKITARM=$DEVKITARM" >> ~/.bashrc
+
+echo "if [ -f ~/.bashrc ]; then . ~/.bashrc; fi" >> ~/.bash_profile
+```
+
+### Choosing where to store pokeemerald (macOS)
+At this point, you can choose a folder to store pokeemerald into. If so, you'll have to change the **current working directory** every time you open the Terminal. If you're okay with storing pokeemerald in the user folder, then proceed to [Installation](#installation).
+
+For example, if you want to store pokeemerald (and agbcc) in **~/Desktop/decomps**, enter this command:
+```bash
+cd Desktop/decomps
+```
+Note that the directory **must exist** in the folder system. If you want to store pokeemerald in a dedicated folder that doesn't exist (e.g. the example provided above), then create the folder (e.g. using Finder) before executing the `cd` command.
+
+(If the path has spaces, then the path must be wrapped with quotations, e.g. `cd "Desktop/decomp folder"`)
+
+If this works, then proceed to [Installation](#Installation). Otherwise, ask for help on IRC or Discord (see [README.md](README.md)).
+
+## Linux
+Open Terminal and enter the following commands, depending on which distro you're using.
+
+### Debian/Ubuntu-based distributions
+Run the following command to install the necessary packages:
+```bash
+sudo apt install build-essential binutils-arm-none-eabi git libpng-dev
+```
+Then proceed to [Choosing where to store pokeemerald (Linux)](#choosing-where-to-store-pokeemerald-linux).
+
+### Other distributions
+_(Specific instructions for other distributions would be greatly appreciated!)_
+
+Try to find the required software in its repositories:
+- `gcc`
+- `g++`
+- `make`
+- `git`
+- `libpng-dev`
+
+Then, follow the instructions [here](https://devkitpro.org/wiki/devkitPro_pacman) to install devkitPro's pacman installer. As a reminder, the goal is to configure an existing pacman installation to recognize devkitPro's repositories.
+
+Once devkitPro pacman is configured, run the following commands:
+```bash
+sudo pacman -Sy
+sudo pacman -S gba-dev
+```
+Note: the last command will ask for the selection of packages to install. Just press Enter to install all of them, followed by entering Y to proceed with the installation.
```

Comment:
The macOS instructions are missing a similar note about `sudo dkp-pacman -S gba-dev`